packages/core/src/api/toDecimal.ts

```diff
@@ -31,7 +32,9 @@ export function toDecimal<TAmount, TOutput>(calculator: Calculator<TAmount>) {
     // eslint-disable-next-line functional/no-expression-statement
     assert(isDecimal, NON_DECIMAL_CURRENCY_MESSAGE);
 
-    const units = toUnitsFn(dineroObject);
+    const units = toUnitsFn(dineroObject).map((unit, index) => {
+      return index === 0 ? unit : absoluteFn(unit);
+    });
```

Comment:
We could do this in `getDecimal` instead, where we already map over units. This also gathers all the decimal-related transforms in one place.

Reply:
I started with that approach, but then I realized I'd need to supply `getDecimal` with the `absoluteFn` and then took the above approach instead.

... but I agree it would be nicer to consolidate the transformation, so I'll adjust.